help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -1420,13 +1420,37 @@
 
       <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
 
-      <h4>Window to front<span class="since">since 5.7.4</span></h4>
+      <h4>Window management<span class="since">since 5.7.6</span></h4>
+
+      <p>JMRI uses a lot of windows.  Each panel is a window, various tools have windows, tables
+      can be shown in windows, etc.  The <strong>Window management</strong> action provides
+      three tools to help manage the windows.</p>
+
+      <dl>
+        <dt>Show/Hide</dt>
+        <dd>The <strong>visibility</strong> of the window is changed.  The window size and location
+        are not affected.</dd>
+
+        <dt>Maximize/Normalize/Minimize</dt>
+        <dd>Minimize and normalize use the OS process to move the window to and from the task bar or
+        dock.  Normalize returns the window to the same location and size.  Maximise fits the
```

Comment:
No need to change, but I noticed that you have the word `Maximise` (with s instead of z) here, but `maximized` later. But it probably doesn't matter.